src/main/java/org/apache/commons/lang3/ThreadUtils.java

```diff
@@ -292,12 +292,12 @@ private AlwaysTruePredicate() {
         }
 
         @Override
-        public boolean test(@SuppressWarnings("unused") final ThreadGroup threadGroup) {
```

Comment:
No. But if I can make Eclipse users a litte bit more happy, I'll merge this for you :-)